### PR TITLE
results panel layout refactor

### PR DESCRIPTION
**Pull request by Gnafu**
Changed the results panel style

### CODE REVIEW

File: web/client/components/data/featuregrid/FeatureGrid.jsx [hunk 1]
```diff
@@ -179,40 +182,50 @@ const FeatureGrid = React.createClass({
...
-                        next: '>',
-                        last: '>|',
-                        first: '|<',
-                        previous: '<'}}
-                    onGridReady={this.onGridReady}
-                    {...this.props.agGridOptions}
-                />
-            </div>
+            <div style={{
+                display: "flex",
```
Comment (mbarto) [inline web/client/components/data/featuregrid/FeatureGrid.jsx]: Is this working on IE9?

Comment (offtherailz) [inline web/client/components/data/featuregrid/FeatureGrid.jsx]: Can we keep that for now?

**Comment by offtherailz** [comment]
Should we solve codacy issues before to merge?

**Comment by Gnafu** [comment]
@offtherailz yes, I'm working on it

**Pull request merged**